frontend/lib/fhirUtils.ts

```diff
@@ -93,7 +93,9 @@ export const getTask = (serviceRequest: ServiceRequest, primaryCondition: Condit
 
         //TODO: Not setting this, made by CPS - but should maybe set it for existing
         basedOn: carePlanRef && [carePlanRef],
-        for: serviceRequest.subject,
+        for: {
+            reference: patient.id
+        },
```

Comment:
The CPS won't understand this relative reference, as it doesn't have the `Patient` resource, and it isn't sent. It might work due to not having referential integrity checks enabled. 

I think this function should be rewritten so that it sends out a `Bundle` to the CPS that is according to the spec: https://santeonnl.github.io/shared-care-planning/Bundle-cps-bundle-01.json

So in this case, the ServiceRequest also doesn't have to be "cleaned up", but all relative reference need to be replaced with bundle references. This way, the CPS can copy the resources, and assign their own `id` values.

Reply:
I already had a feeling this would be too easy ;)